Makefile

```diff
@@ -32,7 +32,7 @@ JEKYLL_OPTS := -d '$(SITE_DESTDIR)' $(if $(SITE_BASEURL),-b '$(SITE_BASEURL)',)
 
 VERSION := $(shell git describe --tags --dirty --always)
 
-IMAGE_REGISTRY ?= k8s.gcr.io/nfd
+IMAGE_REGISTRY ?= quay.io/zvonkok
```

Comment:
??

Reply:
Zvonko secretly moving us to a new world

Reply:
I'd suggest to change this back to `k8s.gcr.io/nfd` 😉